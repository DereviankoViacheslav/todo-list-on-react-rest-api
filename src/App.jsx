import React from 'react';
import './App.scss';

function App() {
  return (
    <div>Todo List</div>
  );
}

export default App;