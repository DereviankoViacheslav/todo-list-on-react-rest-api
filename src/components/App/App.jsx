import React from 'react';
import TodoList from '../TodoList';
import './App.scss';

function App() {
  return (
    <TodoList />
  );
}

export default App;