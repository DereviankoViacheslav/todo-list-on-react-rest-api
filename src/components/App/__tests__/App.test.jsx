import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';
 
describe('<App />', () => {
  it('should display TodoList', () => {
    const wrappedComponent = shallow(<App />);
    expect(wrappedComponent).toMatchSnapshot();
  });
  
  it('should display TodoList', () => {
    const wrappedComponent = shallow(<App />);
    expect(wrappedComponent.find('TodoList').exists()).toBeTruthy();
  });
});