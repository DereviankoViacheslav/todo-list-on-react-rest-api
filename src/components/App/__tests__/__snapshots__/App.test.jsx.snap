// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should display TodoList 1`] = `<App />`;
